app/views/application/_navbar.html.erb

```diff
@@ -51,8 +51,8 @@
               <%= render 'application/menus/user_list_link' %>
               <%= render 'application/menus/offers_dashboard_link' %>
               <%= render 'application/menus/inquiries_list_link' %>
-              <%= render 'application/menus/offers_by_tag_link' %>
               <% if current_user.manages? current_organization %>
+                <%= render 'application/menus/offers_by_tag_link' %>
```

Comment:
is this now only for admins?

Reply:
ping @mllocs 

cc/ @sseerrggii

Reply:
During the last redesign conversations we had it was proposed to remove this section (to simplify the interface). Instead, I'm making it available only for admins.